src/main/java/frc/robot/subsystems/lifter/LifterComponents.java

```diff
@@ -0,0 +1,15 @@
+package frc.robot.subsystems.lifter;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.robot.hardware.digitalinput.IDigitalInput;
+import frc.robot.hardware.motor.ControllableMotor;
+import frc.robot.hardware.signal.InputSignal;
+
+public record LifterComponents(
```

Comment:
```suggestion
public record LifterStuff(
```
for now this is the convention